README.md

```diff
@@ -1,3 +1,4 @@
+<p align="center"><img src="https://raw.githubusercontent.com/jbeguna04/algorithms/master/LogoDesins/128pxblue.png" /></p>
 <p align="center"><img width="40%" src="docs/source/_static/algorithms_logo.png" /></p>
```

Comment:
Please also replace the typography too, with the one that was finalized in #369. Thanks.